packages/marathon/build

```diff
@@ -19,6 +19,8 @@ StartLimitInterval=0
 RestartSec=15
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/marathon
```

Comment:
Ok, well most of the environment variables I want to set (e.g. SSL_) are ready by libmesos directly.